techeria_controller/templates/registration.html

```diff
@@ -38,49 +38,54 @@ <h3>Sign UP</h3>
 <form action="registration" method="POST">
 
   {% csrf_token %}
-
+  <div style="text-align: center;">
+    {% for message in messages %}
+    <h4 style="color: red;">{{message}}</h4>
+    {% endfor %}
+  </div>
+  
 <table align="center" cellpadding = "10">
 
 <!----- First Name ---------------------------------------------------------->
 <tr>
 <td>FIRST NAME</td>
-<td><input type="text" name="First_Name" maxlength="30"/>
+<td><input type="text" name="First_Name" maxlength="30" required/>
 (max 30 characters a-z and A-Z)
 </td>
 </tr>
 
 <!----- Last Name ---------------------------------------------------------->
 <tr>
 <td>LAST NAME</td>
-<td><input type="text" name="Last_Name" maxlength="30"/>
+<td><input type="text" name="Last_Name" maxlength="30" required/>
 (max 30 characters a-z and A-Z)
 </td>
 </tr>
 <tr>
   <td>USER NAME</td>
-  <td><input type="text" name="Username" maxlength="30"/>
+  <td><input type="text" name="Username" maxlength="30" required/>
   (max 30 characters a-z and A-Z)
   </td>
   </tr>
 
 <!----- Date Of Birth -------------------------------------------------------->
 <tr>
   <td>Date of Birth</td>
-  <td><input type="Date" name="Date_of_Birth"/>
```

Comment:
We can make a check to make sure the date of birth is a date that would prove that the person signing up or signing in is not a child.